client/src/Components/Mentors.js

```diff
@@ -1,143 +1,138 @@
 import React, { useEffect, useState } from "react";
-import {
-	BrowserRouter as Router,
-	Switch,
-	Route,
-	Link,
-} from "react-router-dom";
+import dayjs from "dayjs";
 import "../App.css";
 import "../grid.css";
 
 export default function Mentors(props) {
-	const [newQuizzQuestions, setNewQuizzQuestions]=useState([]);
-	const [refreshQuestions, setRefreshQuestions]=useState(false);
-
-	const [newQuizz,setNewQuizz]=useState(	{
-
+	const [newQuizzQuestions, setNewQuizzQuestions] = useState([]);
```

Comment:
I know this isn't your change, but this still needs to be updated for spelling. `newQuizQuestions, setNewQuizQuestions`

Reply:
I tried to changed it but it was so confusing, I'll try again